### PR TITLE
Feature/asset alert

### PR DESCRIPTION
**Pull request by SNuseibeh**
## Checklist:

- [ ] I have updated the app changelog
- [ ] I have made corresponding changes to the documentation
- [ ] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have performed a self-review of my own code
- [ ] My changes generate no new warnings on the linter

### CODE REVIEW

File: src/components/app/alerts/createAlert.tsx [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+}
+
+ const CreateAlert = ({width}: AlertProps) => {
+    const { t } = useTranslation(["alert"]);
+    const connectedExchanges = useSelector(selectConnectedExchanges);
+    const [loading, setLoading] = useState(false);
+
+    useEffect(() => {
+        connectedExchanges?.map((elm) => elm?.provider_id != null ? _exchanges.push(elm.name) : null)
+    }, [connectedExchanges])
```
Comment (almujtabamohammed) [inline src/components/app/alerts/createAlert.tsx:26]: `connectedExchanges` and `_exchanges` are not used anywhere, please remove if they are not needed

File: src/components/shared/dropdowns/assets.tsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+          <DropdownMenu.Portal className="z-10">
+            <DropdownMenu.Content
+              className={clsx(
+                " min-w-[500px] max-h-[400px] dark:bg-offwhite-1 bg-grey-1 w-96 rounded-md z-50"
+              )}
+   
+            >
+    
+              <Col>
+              <p className="dark:text-black-1 text-white">Select Asset</p>
```
Comment (almujtabamohammed) [inline src/components/shared/dropdowns/assets.tsx:43]: Add translation

File: src/pages/settings/index.tsx [hunk 3]
```diff
@@ -38,6 +39,7 @@ const SettingsScreen = () => {
                             <Tab className="font-bold text-lg outline-none cursor-pointer flex-1 text-center md:flex-none md:px-10" onClick={onClickTab}>{t('account')}</Tab>
                             <Tab className="font-bold text-lg outline-none cursor-pointer flex-1 text-center md:flex-none md:px-10" onClick={onClickTab}>{t('exchange')}</Tab>
                             <Tab className="font-bold text-lg outline-none cursor-pointer flex-1 text-center md:flex-none md:px-10" onClick={onClickTab}>{t('subscription')}</Tab>
+                            <Tab className="font-bold text-lg outline-none cursor-pointer flex-1 text-center md:flex-none md:px-10" onClick={onClickTab}>{t('alerts')}</Tab>
```
Comment (almujtabamohammed) [inline src/pages/settings/index.tsx:42]: You need to add "alerts" to settings translation file

File: src/components/app/alerts/createAlert.tsx (hunk 1 above)
Comment (SNuseibeh) [reply src/components/app/alerts/createAlert.tsx]: _exchanges is being used to add predefined values in the input.

File: src/components/app/alerts/alertsTable.tsx [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+    })()
+    }, [])
+
+    const renderStatus = (condition: string) => {
+      if(condition === 'LESS_THAN') return t("drops_to")
+      return t("price_reaches")
+    }
+
+    return  <Col className="flex flex-1 gap-10 w-full mt-4">
+        <table className={twMerge(styles.table,'dark:text-white')}>
```
[the remark this answers is not in the document]
Comment (almujtabamohammed) [reply src/components/app/alerts/alertsTable.tsx:44]: @SNuseibeh we need to use this component in all tables to have consistent design and sorting features

**Review by almujtabamohammed — APPROVED** [review]

**Pull request merged**